tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es6",
-    "module": "commonjs",
+    "target": "es2021",
```

Comment:
this seems like a bigger change, was it related to the linting?

Reply:
actually I wanted to upgrade also these configs but at the end of the day I realized that was breaking a test, so reverted the change to how it was before